src/controllers/user.controllers.js

```diff
@@ -0,0 +1,41 @@
+import User from '../models/user';
+import { registerValidation, loginValidation } from '../routes/user/validation';
+import { StatusCodes } from 'http-status-codes';
+import bcrypt from 'bcryptjs';
+
+export const createUser = async (data, response) => {
+  const { error } = registerValidation(data);
+  if (error) return response.status(StatusCodes.BAD_REQUEST).send(error.details[0].message);
+
+  const emailExist = await User.findOne({ email: data.email });
```

Comment:
This is well handled but consider also this: https://masteringjs.io/tutorials/mongoose/unique - checking for uniqueness could be centralised in the user schema and possibly also in the validation schema for user instances and used when a user instance is edited.